frontend/packages/jobs/trigger.config.ts

```diff
@@ -33,7 +36,30 @@ export default defineConfig({
       prismaExtension({
         schema: '../../packages/db/prisma/schema.prisma',
       }),
+      esbuildPlugin(
+        sentryEsbuildPlugin({
+          org: 'liam-hq',
+          project: 'trigger-jobs',
+          authToken: process.env.SENTRY_AUTH_TOKEN,
+        }),
+        { placement: 'last', target: 'deploy' },
+      ),
     ],
   },
+  init: async () => {
+    Sentry.init({
+      dsn: process.env.SENTRY_DSN,
+      environment:
+        process.env.NODE_ENV === 'production' ? 'production' : 'development',
```

Comment:
Please following setup:

https://github.com/liam-hq/liam/blob/f78549742f09b14067bbef782cc1f9180a0ed350/frontend/apps/app/sentry.client.config.ts#L27

Reply:
Please use `NEXT_PUBLIC_ENV_NAME` in environment.